packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -364,6 +364,8 @@ export default {
                 return
             }
 
+            axios.defaults.headers.common['X-Cert-Analytics-Dt'] = true
```

Comment:
Not an js expert but will it get converted to string "true"?

Reply:
In my tests it was converted to a string.